src/julia-parser.scm

```diff
@@ -94,9 +87,17 @@
 (define ctrans-op (string->symbol "'"))
 (define vararg-op (string->symbol "..."))
 
-(define operators (list* '~ '! '¬ '-> '√ '∛ '∜ ctrans-op trans-op vararg-op
-                         (delete-duplicates
-                          (apply append (map eval prec-names)))))
+(define (is-word-operator? op)
+  (let loop ((chars (string->list (symbol->string op))))
+    (if (null? chars)
+         #t
+         (and (identifier-start-char? (car chars)) (loop (cdr chars))))))
```

Comment:
Yes, it is. I was looking for that! In Racket, the `every` function is usually spelt `andmap`.